### DISCUSSION TITLE
If there are multiple Renovate PR open, is there a way to stagger the updates from main?

### DISCUSSION THREAD

**Discussion by tomwillis608** [renovatebot/renovate#16948] [Request Help]
We like Renovate and are making quite a bit of use.  We have taken several noise reduction step per the suggestions in the docs.  
We have a number of GitHub actions that we run on each PR and each change on a PR. 
If there are a number of PRs that Renovate has opened, when it detects a change in main and updates the PRs, it seems to do all the PRs at the same time, causing all the actions to run on each PR at once.
Is there a configuration that would let Renovate wait a bit between updating its open PRs?  
TIA

**Comment by Gabriel-Ladzaretti** [comment]
You could try using [Dependency Dashboard Approval](https://docs.renovatebot.com/getting-started/use-cases/#dependency-dashboard-approval)
or [prConcurrentLimit](https://docs.renovatebot.com/configuration-options/#prconcurrentlimit) to limit the concurrent renovate PRs.

[prCreation](https://docs.renovatebot.com/configuration-options/#prcreation) might help as well.
